eslint.config.js

```diff
@@ -10,6 +10,102 @@ export default [
     languageOptions: {
       globals: {
         ...globals.browser,
+        //PrimeVue Components
```

Comment:
Why is the configuration of global variables better than the local import in each component? Isn't that exactly what using no-undef-components rule is supposed to do? 

e.g. https://github.com/remsfal/remsfal-frontend/blob/db2d42436e453ef6994a7eef79055a790b638dba/src/components/NewProjectForm.vue#L3